pkg/jobrunaggregator/tablescreator/table_creator.go

```diff
@@ -48,5 +48,19 @@ func (r *allJobsTableCreatorOptions) Run(ctx context.Context) error {
 		fmt.Fprintf(os.Stdout, "table already exists: %s\n", jobrunaggregatorlib.JobRunTableName)
 	}
 
+	// Create TestRunsTable
+	testRunsTable := r.ciDataSet.Table(jobrunaggregatorlib.TestRunTableName)
```

Comment:
Consider a structure like

```go
TableNamesToSchemas := map[string]string{
  jobrunaggregatorlib.TestRunTableName: jobrunaggregatorapi.TestRunsSchema,
  jobrunaggregatorlib.JobRunTableName: jobrunaggregatorapi.JobRunSchema
}

```

And then this command iterates over it creating each table.

Reply:
that works out really well -- thanks!